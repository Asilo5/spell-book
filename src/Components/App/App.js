import React, { Component } from 'react';
import HouseForm  from '../../Containers/HouseForm/HouseForm';
import  SpellsContainer  from '../../Containers/SpellsContainer/SpellsContainer';
import FavouritesContainer from '../../Containers/FavouritesContainer/FavouritesContainer';
import { Route } from 'react-router-dom';

import './App.css';

class App extends Component {
  render() {
    return (
      <section>
        <Route exact path='/' render={() => <HouseForm /> } />
        <Route exact path='/spells' render={() => <SpellsContainer /> } />
        <Route exact path='/favourites' render={() => <FavouritesContainer /> } />
      </section>
    );
  }
}

export default App;
