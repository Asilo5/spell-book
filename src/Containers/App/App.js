import React, { Component } from 'react';

import './App.css';

class App extends Component {
  render() {
    return (
      <section>
         <h1>Hello</h1>
      </section>
    );
  }
}

export default App;
