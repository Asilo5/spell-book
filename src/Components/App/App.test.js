import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App', () => {
  it('should match snapshot with correct data passing through', () => {
    let wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  })
})
 