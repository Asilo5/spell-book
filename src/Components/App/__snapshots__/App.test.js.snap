// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot with correct data passing through 1`] = `
<section>
  <Route
    exact={true}
    path="/"
    render={[Function]}
  />
  <Route
    exact={true}
    path="/spells"
    render={[Function]}
  />
  <Route
    exact={true}
    path="/favourites"
    render={[Function]}
  />
</section>
`;
